app/controllers/foreman_tasks/api/tasks_controller.rb

```diff
@@ -27,6 +27,10 @@ def summary
       param :id, :identifier, desc: 'UUID of the task'
       def show; end
 
+      api :GET, '/tasks/:id/details', 'Show task extended details'
+      param :id, :identifier, desc: 'UUID of the task'
+      def details; end
```

Comment:
`details` also needs to be mapped to `view` permission in `action_permission` somewhere down in this file.